content/en/cloud_cost_management/recommendations/_index.md

```diff
@@ -0,0 +1,134 @@
+---
+title: Cloud Cost Recommendations
+kind: documentation
+private: true
+description: Learn how to reduce the spending of your organization's cloud resources with Cost Recommendations.
+further_reading:
+- link: "/cloud_cost_management/"
+  tag: "Documentation"
+  text: "Learn about Cloud Cost Management"
+---
+
+{{< site-region region="gov" >}}
+<div class="alert alert-warning">Cloud Cost Management is not supported for this site.</div>
+{{< /site-region >}}
+
+{{< callout url="#" btn_hidden="true" >}}
+Cloud Cost Recommendations is in public beta with support for AWS, and is automatically enabled if you have set up <a href="/cloud_cost_management/">Cloud Cost Management</a>.
+{{< /callout >}}
+
+## Overview
+
+[Cloud Cost Recommendations][1] provides recommendations on reducing your cloud spending by optimizing the usage of your cloud resources. 
```

Comment:
```suggestion
[Cloud Cost Recommendations][1] provide recommendations on reducing your cloud spending by optimizing the usage of your cloud resources. 
```
Optional, but I think proper nouns can still be treated as plural if it makes grammatical sense to do so